CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/server/util/ZipUtils.java

```diff
@@ -38,4 +45,89 @@ public static void unzip(InputStream in, File destination) throws IOException {
         zis.close();
         in.close();
     }
+
+    public boolean deleteRecursive(File fileOrDirectory) {
+        if (fileOrDirectory.isDirectory()) {
+            for (File child : fileOrDirectory.listFiles()) { deleteRecursive(child); }
+        }
+        return fileOrDirectory.delete() || !fileOrDirectory.exists();
+    }
+
+    public void zipDirectory(String srcDirPath, File zipFile) {
+        List<String> zipFiles = new ArrayList<>();
```

Comment:
I provided comments while reviewing Blake's PR pull/1631, and he made changes. Please make sure your PR includes ALL his changes after review. Such as zipFile, is defined as a param here but also a method in this class, using it this way cause confusion.

Reply:
I repeat: why is someone copying over my changes???  Why not just merge the PR I proposed??  Do you not like the way I type or something? ;-P

Reply:
Blake,  we thought of having one PR first so that it can be tested with python changes and can have one single PR rather than have 4 PRs for one purpose. Due to copy and paste, looks like there are some missing changes. Anyways, we decided to test with python logic and have only python logic on this PR and merge 3 PRs for all 3 platforms separately.

Reply:
@bmeike, We will first merge all the dev changes only. I created this PR as I didn't wanted to mess with your code and as Sri pointed out we want all the changes in one Place.

Also, as I've done few additional changes to keep the zip log file consistent, which is not available in neither your nor Sandy's branch.